goat.sh

```diff
@@ -7,6 +7,16 @@ errcho(){
     >&2 echo $@
 }
 
+# Check for $EDITOR and editor in the path
+if [ -n "$EDITOR" ]; then
+    EDITOR_COMMAND="$EDITOR"
+elif command -v editor >/dev/null 2>&1; then
```

Comment:
At least all of the debian-based distros provide an editor command that is pointed to whatever the user has configured as the default editor, which seemed prominent enough to be worth checking.